types/unist-util-remove-position-test.ts

```diff
@@ -0,0 +1,21 @@
+import {Node} from 'unist'
+
+import remark = require('remark')
+
+import removePosition = require('unist-util-remove-position')
+
+const tree: Node = remark().parse(
+  'Some _emphasis_, **importance**, and `code`.'
+)
+
+remark()
```

Comment:
huh, `xo` is unhappy on Travis

```
types/unist-util-remove-position-test.ts:11:1

  ✖  11:1  Promises must be handled appropriately or explicitly marked as ignored with the void operator.  @typescript-eslint/no-floating-promises
```

might need to tweak this more to

```suggestion
await remark()
```

Reply:
Or using `processSync`

Reply:
I likes adding 'types/' to xo.ignore in package.json, I think xo is often too strict, especially for ts